assets/Icons/optimized/attach.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="320" height="320" viewBox="0 0 320 320" xmlns="http://www.w3.org/2000/svg">
```

Comment:
are these the new icons from Manuel? if so these should all use 240 instead of 320

Reply:
seems like these icons are out of date?

Reply:
I mentioned this below but we don't have the new ones yet.